setup.sh

```diff
@@ -20,12 +20,12 @@ touch ~/.sync-music/config/keys.json
 
 echo -e "\nGiving executable permissions..."
 chmod +x ./src/generate_temp_files.sh
-chmod +x ./src/sync_music.py
+chmod +x ./sync_music.py
 
 echo -e "\nCreating required symlinks..."
 if [ ! -L ~/.sync-music/scripts/generate_temp_files.sh ]; then
     ln -s ./src/generate_temp_files.sh ~/.sync-music/scripts/generate_temp_files.sh 
 fi
 if [ ! -f $HOME/bin/sync-music ]; then 
-    ln ./src/sync_music.py $HOME/bin/sync-music
+    ln -s ./src/sync_music.py $HOME/bin/sync-music
```

Comment:
I believe "sync_music.py" is outside of "./src/", so it won't be able to create the link. And any specific reason for making it soft link?

Reply:
Yup. We can't use hard links. Since, then it tries to find src folder in ~/bin/ . If we use a symbolic link, they redirect it to the correct folder where src is present too.
Of course we can consider adding to PATH variable, but that'd be an overkill. 
Your views.

Reply:
Yeah, that's great.